### PR TITLE
fr-FR

### PR DESCRIPTION
**Pull request by boscorelly**
fr-FR

### CODE REVIEW

File: Files/MultilingualResources/Files.fr-FR.xlf [hunk 1]
```diff
@@ -1,8 +1,8 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
   <file datatype="xml" source-language="en-US" target-language="fr-FR" original="FILES/STRINGS/EN-US/RESOURCES.RESW" tool-id="MultilingualAppToolkit" product-name="n/a" product-version="n/a" build-num="n/a">
     <header>
-      <tool tool-id="MultilingualAppToolkit" tool-name="Multilingual App Toolkit" tool-version="4.0.6915.0" tool-company="Microsoft" />
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.fr-FR.xlf:5]: Can you please fix the spacing here?

**Review by yair100 — CHANGES_REQUESTED** [review]

File: Files/MultilingualResources/Files.fr-FR.xlf (hunk 1 above)
Comment (boscorelly) [reply Files/MultilingualResources/Files.fr-FR.xlf]: done :)

**Review by tsvietOK — APPROVED** [review]

**Review by tsvietOK — CHANGES_REQUESTED** [review]
Resolve conflicts, please.

File: Files/MultilingualResources/Files.fr-FR.xlf [hunk 2]
```diff
@@ -2015,6 +2035,14 @@
           <source>Disconnect</source>
           <target state="translated">Déconnecter</target>
         </trans-unit>
+        <trans-unit id="PreemptiveCacheParallelLimit.Header" translate="yes" xml:space="preserve">
```
Comment (gave92) [inline Files/MultilingualResources/Files.fr-FR.xlf:2038]: I think something went wrong, this string was removed long ago

**Comment by esibruti** [comment]
@gave92 @yair100 Is there a possibility to take the work already done by him and finish?

File: Files/MultilingualResources/Files.fr-FR.xlf (hunk 2 above)
Comment (boscorelly) [reply Files/MultilingualResources/Files.fr-FR.xlf:2038]: mmmh ? im going to copy the entire actual french manually from the repo...

**Comment by yaichenbaum** [comment]
Closing due to inactivity.

**Pull request closed without merging**